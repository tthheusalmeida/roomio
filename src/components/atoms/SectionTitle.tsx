export default function SectionTitle({ title }: { title: string }) {
  return <h2 className="text-lg">{title}</h2>;
}
