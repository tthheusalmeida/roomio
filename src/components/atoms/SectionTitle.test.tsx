import { render, screen } from "@testing-library/react";
import SectionTitle from "./SectionTitle";

describe("SectionTitle component", () => {
  it("renders the title text correctly", () => {
    const { asFragment } = render(<SectionTitle title="My Section" />);

    const heading = screen.getByText("My Section");
    expect(heading).toBeInTheDocument();
    expect(heading.tagName).toBe("H2");

    expect(asFragment()).toMatchSnapshot();
  });
});
