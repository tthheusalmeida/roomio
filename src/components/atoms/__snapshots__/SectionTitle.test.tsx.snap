// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`SectionTitle component renders the title text correctly 1`] = `
<DocumentFragment>
  <h2
    class="text-lg"
  >
    My Section
  </h2>
</DocumentFragment>
`;
